sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/CatalystTypeConverters.scala

```diff
@@ -421,6 +421,17 @@ object CatalystTypeConverters {
     case other => other
   }
 
+  def convertToCatalystUnsafe(a: Any): Any = a match {
```

Comment:
Add a comment like `convertToCatalyst`.

This is just for test and its function doesn't really convert all Scala objects to unsafe Catalyst rows/types. We don't want developers to think they can use it for the purpose the function name suggesting.